Tasks/Yudin/Task1/Task1.py

```diff
@@ -0,0 +1,8 @@
+
+deposit = 20000
+percent = 0.15
+duration = 60
```

Comment:
these values should be of the decimal data type, since they are money-related, -1 point for the unwise choice